libs/ngx-localstorage/src/lib/interfaces/storage-configuration.ts

```diff
@@ -14,4 +14,8 @@ export interface NgxLocalstorageConfiguration {
      * Determines the storage type. (Default: localStorage)
      */
     storage?: Storage
+    /**
+     * Determines the delimiter, will used befoure key. (Default: _)
+     */
+     delimiter?: string
```

Comment:
When extending the configuration the extension should be reflected in the documentation.

Furthermore I'd suggest adding the default value `_` to the `defaultConfig` object exported in `utils.ts`

Reply:
Fixed